packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -43,8 +43,16 @@ const BLOCK_THRESHHOLDS = {
   'eip155:1': 1000000000, //mainnet
   'eip155:3': 1000000000, //ropsten
   'eip155:5': 1000000000, //goerli
+  'eip155:100': 1000000000, //gnosis
   'eip155:1337': 1, //ganache
 }
+const ANCHOR_AUTHORS = {
```

Comment:
are these the addresses of the smart contracts?  Can we call this ANCHOR_CONTRACT_ADDRESSES?

Reply:
+1 to @stbrody 

Also, are they really going to have the same address on all networks?
I assume we will update these addresses when we redeploy the contract with the new function signature!

Reply:
no they won't. I will update the issues+comment for the block numbers to also include adding the correct contract addresses